### PR TITLE
Tutorial for using IDEA to develop UnetAgents

### PR DESCRIPTION
**Pull request by shortstheory**
I've written a tutorial replete with screenshots on how to use IntelliJ IDEA for developing UnetAgents. Please do let me know if anything needs changing. I have not found a `feature-img` and `thumbnail` for this post, so suggestions for the same will also be welcome!

### CODE REVIEW

**Comment by manuignatius** [comment]
You can use https://unsplash.com/ to find a suitable `feature-img` and `thumbnail` for the post @shortstheory

**Comment by shortstheory** [comment]
Thank you for the feedback! I have added a suitable image and fixed the typo,

File: _posts/2019-04-26-using-idea-with-unetstack.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+## Writing the Agent
+
+Create a new Groovy in the `src/` directory of your project by right-clicking the `src` directory in Project pane in IDEA.
+
+![](assets/img/idea-setup/project.png)
+
+In this example, the agent we're writing will be called `AwesomeAgent`. The code for the same is below. For more information about writing agents, refer to the Unet [documentation](https://www.unetstack.net/unet-agents.html).
+
+```
```
Comment (notthetup) [inline _posts/2019-04-26-using-idea-with-unetstack.md]: Can we add language hint to this (and all other) fenced code block?? I believe our markdown renderer will use that for automatic syntax highlighting. https://jekyllrb.com/docs/configuration/markdown/

Would look something like this

```
```java
```

Comment (shortstheory) [reply _posts/2019-04-26-using-idea-with-unetstack.md]: Thanks! I have added this hint.

**Comment by notthetup** [comment]
Merging. Thanks @shortstheory!

**Comment by shortstheory** [comment]
Thanks! Happy to help :-)

**Pull request merged**